.gitignore

```diff
@@ -0,0 +1 @@
+games.log
```

Comment:
@jesrodri configure seu editor para adicionar EOF(End of file) pode acontecer de isso causar algum problema.